File: Iniciante/1048/1048.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    Double value = Double.parseDouble(scanner.nextLine());

    Double adjust = 0.04;

    if (value <= 400) {
      adjust = 0.15;
    } else if (value <= 800) {
      adjust = 0.12;
    } else if (value <= 1200) {
      adjust = 0.1;
    } else if (value <= 2000) {
      adjust = 0.07;
    }

    System.out.printf("Novo salario: %.2f\n", value * (adjust + 1));
    System.out.printf("Reajuste ganho: %.2f\n", value * adjust);
    System.out.printf("Em percentual: %.0f %%\n", adjust * 100);
  }
}

File: Iniciante/1048/1048.rs
use std::io;

fn main() {
  let mut input = String::new();

  io::stdin().read_line(&mut input).unwrap();

  let value = input.trim().parse::<f64>().unwrap();

  let increment = match value {
    0.0..=400.00 => 0.15,
    400.00..=800.00 => 0.12,
    800.00..=1200.00 => 0.1,
    1200.00..=2000.00 => 0.07,
    _ => 0.04,
  };


  println!("Novo salario: {:.2}
Reajuste ganho: {:.2}
Em percentual: {:.0} %", value * (1.0 + increment), value * increment, increment * 100.0);
}